### PR TITLE
Add a source code repo website

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ Table of Contents
   * [codebasehq.com](https://www.codebasehq.com/) — One free project with 100 MB space and 2 users
   * [codeberg.org](https://codeberg.org/) - Unlimited public and private Git repos
   * [gitea.com](https://www.gitea.com/) - Unlimited public and private Git repos
+  * [gitee.com[(https://gitee.com/) - Unlimited public and private Git repos
```
Comment (php-coder) [inline README.md]:
```suggestion
  * [gitee.com](https://gitee.com/) - Unlimited public and private Git repos
```

Comment (LinuxSuRen) [reply README.md]: Thanks for your time.

**Comment by ripienaar** [comment]
I am sure its nice and all but a lot of this is in I think Chinese so it's not widely applicable. I can't imagine any sane non chinese company putting anything here, so I wont be accepting this thank you

**Comment by LinuxSuRen** [comment]
This is free and it has the English UI, I am sure the individual developer can use it.

**Pull request closed without merging**